packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -71,20 +88,31 @@ export class Span implements api.Span, ReadableSpan {
     parentTracer: Tracer,
     context: Context,
     spanName: string,
-    spanContext: api.SpanContext,
-    kind: api.SpanKind,
+    spanContext: SpanContext,
+    kind: SpanKind,
     parentSpanId?: string,
-    links: api.Link[] = [],
-    startTime?: api.TimeInput,
-    clock: Clock = otperformance,
+    links: Link[] = [],
+    startTime?: TimeInput,
+    _deprecatedClock?: unknown, // keeping this argument even though it is unused to ensure backwards compatibility
   ) {
-    this._clock = clock;
     this.name = spanName;
     this._spanContext = spanContext;
     this.parentSpanId = parentSpanId;
     this.kind = kind;
     this.links = links;
-    this.startTime = timeInputToHrTime(startTime ?? clock.now());
+
+    const now = Date.now();
```

Comment:
I think `AnchoredClock` can still be applied here to get a high-resolution start time by default.